oneflow/api/python/functional/python_return_types.h

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This code is referenced from:
+// https://github.com/pytorch/pytorch/blob/master/torch/csrc/utils/structseq.cpp
+
+#ifndef ONEFLOW_API_PYTHON_FUNCTIONAL_PYTHON_RETURN_TYPES_H_
+#define ONEFLOW_API_PYTHON_FUNCTIONAL_PYTHON_RETURN_TYPES_H_
+
+#include <Python.h>
+#include <string>
+#include <sstream>
+#include <structmember.h>
+
+#include "oneflow/api/python/exception/exception.h"
+#include "oneflow/api/python/functional/common.h"
+
+namespace oneflow {
+namespace one {
+namespace functional {
+
+PyObject* returned_structseq_repr(PyStructSequence* obj) {
+  HANDLE_ERRORS
+  PyTypeObject* typ = Py_TYPE(obj);
+  PyObjectPtr tup((PyObject*)obj);
+  if (tup == nullptr) { return nullptr; }
+
+  std::stringstream ss;
+  ss << typ->tp_name << "(\n";
```

Comment:
```suggestion
  ss << tp->tp_name << "(\n";
```

用 `tp` 是不是更好一点。  Linux 的早期的那种省写的风格，其实也是有规律的，并不是简单取前缀。而是省掉元音。
比如 `list` -> `ls`,  `copy` -> `cp`, `move` -> `mv`

然后后面因为越来越复杂，hold 不住了，就改成下划线分隔、全拼（我的看法）。

所以上面的 `typ` ->`tp` 以及 `tup` -> `tuple` 我觉得是不是更好，或者干脆都用全拼（因为也不复杂）